src/main/kotlin/inc/evil/d17map/mappers/Mapper.kt

```diff
@@ -0,0 +1,56 @@
+package inc.evil.d17map.mappers
+
+import inc.evil.d17map.dtos.ClassroomDto
+import inc.evil.d17map.dtos.EquipmentDto
+import inc.evil.d17map.dtos.ReservationDto
+import inc.evil.d17map.dtos.UserDto
+import inc.evil.d17map.entities.Classroom
+import inc.evil.d17map.entities.Reservation
+import org.springframework.stereotype.Component
+
+@Component
```

Comment:
To o co mi tu chodziło to rezygnacja z klasy / obiektu. Zostawiłbym po prostu funkcje same.

Reply:
Poprawione